owl/EASE-OBJ.owl

```diff
@@ -1294,35 +2392,329 @@ It is also possible that a Transient transitionsBack to the initial object. An e
+        </rdfs:subClassOf>
+        <rdfs:comment>A hinged, sliding, or revolving barrier at the entrance to a building, room, or vehicle, or in the frame of a cupboard.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE.owl#Drawer -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#Drawer">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE.owl#DesignedContainer"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A storage compartment without a lid, made to slide horizontally in and out of a piece of furniture.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE.owl#Floor -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#Floor">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#RoomSurface"/>
+        <owl:disjointWith rdf:resource="http://www.ease-crc.org/ont/EASE.owl#Wall"/>
+        <rdfs:comment>The lower interior surface of a room.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE.owl#Gripper -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#Gripper">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE.owl#PrehensileEffector"/>
+        <rdfs:comment>A mechanical device that grasps and holds things.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE.owl#Hand -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#Hand">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE.owl#PrehensileEffector"/>
+        <rdfs:comment>A prehensile effector including palm, fingers, and thumb.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE.owl#Leg -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#Leg">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE.owl#Limb"/>
+        <rdfs:comment>A limb on which an agent walks or stands.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE.owl#Limb -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#Limb">
+        <owl:equivalentClass>
+            <owl:Class>
+                <owl:unionOf rdf:parseType="Collection">
+                    <rdf:Description rdf:about="http://www.ease-crc.org/ont/EASE.owl#Arm"/>
+                    <rdf:Description rdf:about="http://www.ease-crc.org/ont/EASE.owl#Leg"/>
+                </owl:unionOf>
+            </owl:Class>
+        </owl:equivalentClass>
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE.owl#PhysicalEffector"/>
+        <rdfs:comment>An arm or leg of an embodied agent.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE.owl#PrehensileEffector -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#PrehensileEffector">
```

Comment:
huh. This looks oddly familiar. Was it in some other onto file or branch? I remember writing something like this concept once.

Reply:
yes it was somewhere else. maybe EASE-middle